### ISSUE TITLE
Changing line-height and text size does not work properly

### ISSUE THREAD

**Issue by jtlafferty** [edge-reflow/issues#109]
From the customer:
Text formating seems very buggy.. Changing of line height and text size etc does not always work properly.

**Comment by jtlafferty** [comment]
There isn't much action that can be taken on this bug because I can't reproduce in our latest build. If anyone has reproducible steps, please add them to this bug.

**Comment by CoreyDotCom** [comment]
Yeah wish we knew more, clearly line height won't work on a span if it's < a line height set on the element itself or another span... text size seems to work fine.
